src/Job/SyncProject.php

```diff
@@ -193,10 +204,19 @@ public function getScriptoMediaEntity(ScriptoProject $project, Item $item, Media
      *
      * @param Item $item
      * @param ScriptoProject $project
-     * @return array
+     * @return \Doctrine\Common\Collections\ArrayCollection|array
      */
     public function getAllItemMedia(Item $item, ScriptoProject $project)
     {
-        return $item->getMedia();
+        $medias = $item->getMedia();
+        $mediaTypes = $project->getMediaTypes();
+        if ($mediaTypes) {
+            $mediaTypes[] = '';
+            $criteria = \Doctrine\Common\Collections\Criteria::create()
+                ->where(Criteria::expr()->isNull('mediaType'))
```

Comment:
@Daniel-KM We noticed this peculiarity during testing. We're pretty sure a more expected behavior is to _exclude_ media with NULL media type if the project owner explicitly selects media types. Is there a reason you're _including_ them here?

Reply:
It's more related to the database; i saw an error when this criteria is not applied.

Reply:
An error there is alarming. I did not see an error after removing that line. Under what conditions did you see the error? We strongly prefer to exclude NULL media types, so we need to resolve this before we make a release.

Reply:
I don't remember, so it can be skipped.